core/dbt/parser/schemas.py

```diff
@@ -788,6 +788,7 @@ def parse(self) -> List[TestBlock]:
                 refs = ParserRef()
 
             # There's no unique_id on the node yet so cannot add to disabled dict
+            print(f"--- node_plock in parse: {node_block}")
```

Comment:
s/plock/block

Reply:
Oops. Just some testing I did that got left in.